flow/Platform.h

```diff
@@ -406,7 +406,12 @@ dev_t getDeviceId(std::string path);
 #endif
 
 #ifdef __linux__
+#ifndef __aarch64__
 #include <x86intrin.h>
+#else
+// https://github.com/jratcliff63367/sse2neon
+#include <SSE2NEON.h>
```

Comment:
I don't like having this file in `flow`. We already have quite a mess with external libraries in the source code. Can you move this file into the `contrib` directory and then add a target there?

Reply:
Sounds good!